doc/tools/makemd.py

```diff
@@ -0,0 +1,360 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
`makemd.py` was removed for being unused a long time ago. Not sure why it's being added back.